Assets/LeapMotionModules/UI/Interaction/KabschGraspedPose.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.UI.Interaction {
+
+  public class KabschGraspedPose : IGraspedPoseController {
+    public const int NUM_FINGERS = 5;
+    public const int NUM_BONES = 4;
+
+    public enum SolveMethod {
+      SixDegreeSolve,
+      PivotAroundOrigin
+    }
+    public SolveMethod _solveMethod;
```

Comment:
Currently this is overwritten no matter what it is initialized to, so it probably should not be a public field.